.github/scripts/simple-smoke-test.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 set -o errexit
+set -x
```

Comment:
nit - can be move to above line

Reply:
Left over from debugging, removed

276e1ae1f93567526fa180fe3329a4e1834630c3

Reply:
I think this can help with debugging: 
`set -o errexit ${RUNNER_DEBUG:+-x}`

Reply:
separate PR gets my vote
BASH novice question - what does `${RUNNER_DEBUG:+-x}` actually do?

Reply:
> separate PR gets my vote

https://github.com/hazelcast/hazelcast-docker/pull/817

> BASH novice question - what does `${RUNNER_DEBUG:+-x}` actually do?

It expands to `-x` if `RUNNER_DEBUG` is defined as a variable.

Reply:
got it. neat. thanks